presto-cli/src/main/java/com/facebook/presto/cli/StatusPrinter.java

```diff
@@ -193,6 +198,7 @@ public void printFinalInfo()
                     formatDataRate(bytes(stats.getProcessedBytes() / nodes), wallTime, true));
             reprintLine(perNodeSummary);
 
+            // Parallelism: 5.3
```

Comment:
For the future please keep in mind that we usually don't introduce unnecessary changes in commits as it may complicate reviewing and cause confusion.  In this case you can keep this commit as is, since this is a tiny change.